### PR TITLE
release: 4.2.2 changelog

### PR DESCRIPTION
**Pull request by ycjcl868**
## 4.2.2

`2020-05-11`

- 🐞 Fix `npm run version` install error. [#24059](https://github.com/ant-design/ant-design/pull/24059)
- 🐞 Fix Menu `@menu-item-font-size` not working. [#24052](https://github.com/ant-design/ant-design/pull/24052)
- 💄 Add `@modal-close-color` less variable. [#24053](https://github.com/ant-design/ant-design/pull/24053)
## 4.2.2

`2020-05-11`

- 🐞 修复安装 antd `npm run version` 报错的问题。[#24059](https://github.com/ant-design/ant-design/pull/24059)
- 🐞 修复 Menu @menu-item-font-size 变量失效的问题。[#24052](https://github.com/ant-design/ant-design/pull/24052)
- 💄 新增 `@modal-close-color` less 变量。[#24053](https://github.com/ant-design/ant-design/pull/24053)

### CODE REVIEW

**Comment by afc163** [comment]
少个 #24053

File: CHANGELOG.zh-CN.md [hunk 1]
```diff
@@ -15,6 +15,14 @@ timeline: true
 
 ---
 
+## 4.2.2
+
+`2020-05-11`
+
+- 🐞 修复安装 antd `npm run version` 报错的问题。[#24059](https://github.com/ant-design/ant-design/pull/24059)
+- 🐞 修复 Menu @menu-item-font-size 变量失效的问题。[#24052](https://github.com/ant-design/ant-design/pull/24052)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:23]:
```suggestion
- 🐞 修复 Menu `@menu-item-font-size` 变量失效的问题。[#24052](https://github.com/ant-design/ant-design/pull/24052)
```

**Review by zombieJ — APPROVED** [review]

**Review by afc163 — APPROVED** [review]

**Comment by afc163** [comment]
可以发了

**Comment by joesdu** [comment]
怎么还没发,等着升级🤣

**Pull request merged**